NDTensors/test/dense.jl

```diff
@@ -25,83 +26,89 @@ using Test
 
       randn!(A)
 
-      for I in eachindex(A)
-        @test A[I] != 0
-      end
-
-      for I in eachindex(A)
-        @test A[I] != 0
-      end
-
       @test ndims(A) == 2
       @test dims(A) == (3, 4)
       @test inds(A) == (3, 4)
 
-      A[1, 1] = 11
+      Aview = A[2:3, 2:3]
+      @test dims(Aview) == (2, 2)
 
-      @test A[1, 1] == 11
+      B = dev(Tensor(undef, (3, 4)))
+      randn!(B)
+      C = copy(A)
+      C = permutedims!!(C, B, (1, 2), .+)
```

Comment:
Why is this `.+` instead of `+`?

Reply:
Oh let me modify some of these lines. I was trying to debug an issue that had to do with an order of operation in my `allowscalar` portion of the code. This bug has been fixed. I will change this to just + and I will also comment out the `C = A + B` in the `allowscalar` portion of the code. I also added the reason why the code fails.